eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/CassandraHelper.scala

```diff
@@ -34,10 +34,17 @@ object CassandraHelper {
   }
 
 
+  implicit class RowOps(val self: Row) extends AnyVal {
+
+    def wasApplied: Boolean = self.getBool("[applied]")
```

Comment:
`isApplied` ?

Reply:
I named it `wasApplied`, because it is called like that in Cassandra driver, so I thought it might ease discoverability a bit. But `isApplied` sounds fine too.

https://docs.datastax.com/en/drivers/java/3.0/com/datastax/driver/core/ResultSet.html#wasApplied--